sbb_newspapers_org_image/eynollah.py

```diff
@@ -1883,11024 +1276,2424 @@ def do_work_of_slopes_new(self,queue_of_all_params,boxes_text,textline_mask_tot_
+    def write_into_page_xml_full(self, contours, contours_h, page_coord, dir_of_image, order_of_texts, id_of_texts, all_found_texline_polygons, all_found_texline_polygons_h, all_box_coord, all_box_coord_h, found_polygons_text_region_img, found_polygons_tables, found_polygons_drop_capitals, found_polygons_marginals, all_found_texline_polygons_marginals, all_box_coord_marginals, slopes, slopes_marginals):
 
-                arg_neg_must_be_deleted= np.array(range(len(peaks_neg_e)))[y_padded_up_to_down_padded_e[peaks_neg_e]/float(neg_peaks_max)<0.3  ] 
-                diff_arg_neg_must_be_deleted=np.diff(arg_neg_must_be_deleted)
-                
+        found_polygons_text_region = contours
+        found_polygons_text_region_h = contours_h
 
-                
-                arg_diff=np.array(range(len(diff_arg_neg_must_be_deleted)))
-                arg_diff_cluster=arg_diff[diff_arg_neg_must_be_deleted>1]
-                
+        # create the file structure
+        data = ET.Element("PcGts")
 
-                peaks_new=peaks_e[:]
-                peaks_neg_new=peaks_neg_e[:]
-
-                clusters_to_be_deleted=[]
-                if len(arg_diff_cluster)>0:
-                    
-                    clusters_to_be_deleted.append(arg_neg_must_be_deleted[0:arg_diff_cluster[0]+1])
-                    for i in range(len(arg_diff_cluster)-1):
-                        clusters_to_be_deleted.append(arg_neg_must_be_deleted[arg_diff_cluster[i]+1:arg_diff_cluster[i+1]+1])
-                    clusters_to_be_deleted.append(arg_neg_must_be_deleted[arg_diff_cluster[len(arg_diff_cluster)-1]+1:])
-                    
-
-                if len(clusters_to_be_deleted)>0:
-                    peaks_new_extra=[]
-                    for m in range(len(clusters_to_be_deleted)):
-                        min_cluster=np.min(peaks_e[clusters_to_be_deleted[m]])
-                        max_cluster=np.max(peaks_e[clusters_to_be_deleted[m]])
-                        peaks_new_extra.append( int( (min_cluster+max_cluster)/2.0) )
-                        for m1 in range(len(clusters_to_be_deleted[m])):
-                            peaks_new=peaks_new[peaks_new!=peaks_e[clusters_to_be_deleted[m][m1]-1]]
-                            peaks_new=peaks_new[peaks_new!=peaks_e[clusters_to_be_deleted[m][m1]]]
-                            
-                            peaks_neg_new=peaks_neg_new[peaks_neg_new!=peaks_neg_e[clusters_to_be_deleted[m][m1]]]
-                    peaks_new_tot=[]
-                    for i1 in peaks_new:
-                        peaks_new_tot.append(i1)
-                    for i1 in peaks_new_extra:
-                        peaks_new_tot.append(i1)
-                    peaks_new_tot=np.sort(peaks_new_tot)
-                    
-                    
-                else:
-                    peaks_new_tot=peaks_e[:]
+        data.set("xmlns", "http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15")
```

Comment:
Can we update this to PAGE-XML 2019 (http://schema.primaresearch.org/PAGE/gts/pagecontent/2019-07-15)?
(also L1459)